pkg/collector/runner/runner.go

```diff
@@ -58,12 +58,13 @@ type runnerCheckStats struct {
 
 // Runner ...
 type Runner struct {
+	running          uint32                   // Flag to see if the Runner is, well, running
```

Comment:
Could you add a comment somewhere that explains why the `uint32` variables need to be put at the top of structs, so that we do not forget when we go back to this code later?